### PR TITLE
Conv transpose bn fuse pass

### PR DESCRIPTION
**Pull request by fengxiaoshuai**
### PR types
Others

### PR changes
Others

### Describe
test:Conv_transpose_bn_fuse_pass

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_bn_fuse_pass.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                return True
+            return False
+
+        self.add_ignore_check_case(
+            teller1, IgnoreReasons.PASS_ACCURACY_ERROR,
+            "The output format of conv2d_transpose is wrong when data_format attribute is NHWC"
+        )
+
+        self.add_ignore_check_case(teller2, IgnoreReasons.PASS_ACCURACY_ERROR,
+                                   "there is diff when group >1 in this pass")
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_bn_fuse_pass.py:224]: mkldnn下 group>1的时候结果会有diff，还是这个pass在原生下也有diff啊

Comment (fengxiaoshuai) [reply python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_bn_fuse_pass.py:224]: > mkldnn下 group>1的时候结果会有diff，还是这个pass在原生下也有diff啊
都有diff

**Review by jiweibo — DISMISSED** [review]
LGTM

**Review by XieYunshen — DISMISSED** [review]
LGTM for `set_tests_properties(test_conv_transpose_bn_fuse_pass PROPERTIES TIMEOUT 300)`

**Pull request closed without merging**